lxd/api_cluster.go

```diff
@@ -3817,26 +3817,24 @@ func clusterGroupsGet(d *Daemon, r *http.Request) response.Response {
 				return err
 			}
 
-			for i := range clusterGroups {
-				nodeClusterGroups, err := dbCluster.GetNodeClusterGroups(ctx, tx.Tx(), dbCluster.NodeClusterGroupFilter{GroupID: &clusterGroups[i].ID})
+			apiClusterGroups := make([]*api.ClusterGroup, len(clusterGroups))
```

Comment:
please can you use the form `make([]*api.ClusterGroup, 0, len(clusterGroups))` to avoids accidents in the future.

Reply:
Will fix in separate commits